docs/samples/Microsoft.ML.Samples/Dynamic/TensorFlow/TextClassification.cs

```diff
@@ -45,7 +45,7 @@ public static void Example()
             // Load the TensorFlow model once.
             //      - Use it for quering the schema for input and output in the model
             //      - Use it for prediction in the pipeline.
-            var modelInfo = TensorFlowUtils.LoadTensorFlowModel(mlContext, modelLocation);
+            var modelInfo = mlContext.Transforms.LoadTensorFlowModel(modelLocation);
             var schema = modelInfo.GetModelSchema();
             var featuresType = (VectorType)schema["Features"].Type;
```

Comment:
>Features [](start = 51, length = 8)

Can we add a sample that uses `modelInfo.GetInputSchema()` to find out what the name of the input node is?

Reply:
I see its being used at couple of places in the tests e.g.
https://github.com/dotnet/machinelearning/blob/eb959c32a6215b1445df56a49761d065dfcdb7b3/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L894

https://github.com/dotnet/machinelearning/blob/eb959c32a6215b1445df56a49761d065dfcdb7b3/test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs#L849